azure-pipelines.yml

```diff
@@ -72,13 +74,13 @@ jobs:
       architecture: 'x64'
 
   - script: |
-      sudo update-alternatives --set php /usr/bin/php$(phpVersion)
-      sudo update-alternatives --set phar /usr/bin/phar$(phpVersion)
-      sudo update-alternatives --set phpdbg /usr/bin/phpdbg$(phpVersion)
-      sudo update-alternatives --set php-cgi /usr/bin/php-cgi$(phpVersion)
-      sudo update-alternatives --set phar.phar /usr/bin/phar.phar$(phpVersion)
+      sudo update-alternatives --set php /usr/bin/php$(phpver)

```

Comment:
I assume this is because PHP 7.4 is not yet available in the standard repos?

Reply:
yes, as of today :)